report/src/main/webapp/app/layout/toolbar/toolbarTop.view.html

```diff
@@ -19,9 +19,13 @@
 -->
 
 <div class="toolbar-top">
-  <div class="toolbar-toggle">
-    <i class="glyphicon"
-       data-ng-class="sidebarExpanded ? 'glyphicon-chevron-left' : 'glyphicon-chevron-right'"></i>
+  <div class="toolbar-toggle fontawesome-link">
```

Comment:
Tbh i liked the ng-class syntax more - we don't really need this additional markup here